src/backend/joanie/core/models/products.py

```diff
@@ -652,12 +653,19 @@ class Enrollment(BaseModel):
         on_delete=models.RESTRICT,
     )
     is_active = models.BooleanField(
-        help_text="Tick to enroll the user to the course run.",
+        help_text=_("Tick to enroll the user to the course run."),
         verbose_name="is active",
     )
     state = models.CharField(
         _("state"), choices=enums.ENROLLMENT_STATE_CHOICES, max_length=50, blank=True
     )
+    was_created_by_order = models.BooleanField(
+        help_text=_(
+            "Ticked if the enrollment has been initially created in the scope of an order."
```

Comment:
above boolean for `is_active` has help text "Tick to enroll" instead of "Ticked if". Let's homogenize?